exercises/C-comparison-operators/exercise.js

```diff
@@ -10,7 +10,7 @@ let mentorCount = 9;
 let moreStudentsThanMentors = studentCount > mentorCount; // finish this statement
 
 let roomMaxCapacity = 25;
-let enoughSpaceInRoom = moreStudentsThanMentors <= roomMaxCapacity; // finish this statement
+let enoughSpaceInRoom = roomMaxCapacity >= mentorCount + studentCount; // finish this statement
```

Comment:
Yes, you get it. But it will better to enclose the mentorCount + studentCount with a pair of bracket so make it more readable.